pkg/infra/main.go

```diff
@@ -86,11 +92,10 @@ func gameLoop(globalState state.State, agentMap map[commons.ID]agent.Agent, game
 				return
 			}
 		}
-		//todo: fix this
-		//todo: There is a weird bug due to the mathematics that agents gain more health from cowering than monster attack when all cower
+		logging.Log.Infof("------------------------------ Level %d Ended ----------------------------", globalState.CurrentLevel)
```

Comment:
This form of logging won't work anymore because of #53.
Needs to be replaced with:
```suggestion
logging.Log(logging.Info, nil, fmt.Sprintf("------------------------------ Level %d Ended ----------------------------", globalState.CurrentLevel))
```